packages/cli/index.js

```diff
@@ -24,10 +24,6 @@ const liveServer = require('./src/lib/live-server');
 const cliUtil = require('./src/util/cliUtil');
 const logger = require('./src/util/logger');
 
-const {
-  ACCEPTED_COMMANDS,
-  ACCEPTED_COMMANDS_ALIAS,
-} = require('./src/constants');
 const CLI_VERSION = require('./package.json').version;
 
 process.title = 'MarkBind';
```

Comment:
I am not sure what's the purpose of line 29 and also line 30-32. I think I am able to remove them without causing any visual differences. But just to be safe, I will leave them in. If anyone who's reviewing knows what those are for, please let me know 👀

Reply:
Looks like this is to make it easier to identify/manage/log the process in the process manager. I might be wrong but I think we should keep it as it is.

Reply:
Looks like this is to make it easier to identify/manage/log the process in the process manager. I might be wrong but I think we should keep it as it is.